language/de.json

```diff
@@ -1,292 +1,321 @@
 {
-  "semantics": [
+  "semantics":[
     {
-      "fields": [
+      "fields":[
         {
-          "field": {
-            "fields": [
+          "field":{
+            "fields":[
               {
-                "field": {
-                  "fields": [
-                    {},
-                    {},
-                    {},
-                    {},
-                    {},
+                "field":{
+                  "fields":[
                     {
-                      "label": "Kommentare",
-                      "description": "Die Kommentare werden angezeigt, wenn der Nutzer die vorgeschlagenen Antworten für alle Schaubilder anzeigt."
+                      
                     },
                     {
-                      "label": "Kommentare immer anzeigen"
+                      
                     },
                     {
-                      "label": "Undurchlässigkeit des Hintergrunds"
+                      
                     },
                     {
-                      "label": "Als Button anzeigen"
+                      
                     },
                     {
-                      "label": "Titel"
+                      
                     },
                     {
-                      "label": "Gehe zu Schaubild"
+                      "label":"Kommentare",
```

Comment:
@otacke , are you seeing any strange behaviour on your local site by any chance?